observability/logging/logging-6.0/log6x-release-notes.adoc

```diff
@@ -62,9 +62,10 @@ include::snippets/logging-support-exception-for-cluster-observability-operator-d
 
 * This enhancement updates the must-gather functionality to only capture information for inspecting Logging 6.0 components, including Vector deployments from ClusterLogForwarder.observability.openshift.io resources and the Red Hat managed LokiStack. (link:https://issues.redhat.com/browse/LOG-5949[LOG-5949])
 
-
 * This enhancement improves Azure storage secret validation by providing early warnings for specific error conditions. (link:https://issues.redhat.com/browse/LOG-4571[LOG-4571])
 
+* This enhancement updates the `ClusterLogForwarder` API to follow the Kubernetes standards. For example, before this update, the syntax for specifying the delivery mode for log forwarding in the `ClusterLogForwarder` custom resource was `spec.outputs.<output_type>.tuning.delivery`. With this update, the syntax is `spec.outputs.<output_type>.tuning.deliveryMode`. (link:https://issues.redhat.com/browse/LOG-5977[LOG-5977])
```

Comment:
@anpingli, I updated this release note draft, and so I now need one more review from you.

Reply:
@max-cx 
For 5.x       
apiVersion: logging.openshift.io/v1
kind: ClusterLogForwarder
metadata:
  name: <name>
spec:
  outputs:
  - name: <output_name>
    type: <output_type>
    tuning:
      delivery: AtMostOnce
      
For 6.x
apiVersion: "observability.openshift.io/v1"
kind: ClusterLogForwarder
metadata:
  name: <name>
spec:
  outputs:
  - name: <output-name>
    type: <output_type>
    <output_type>:
      tuning:
        deliveryMode: AtMostOnce

Reply:
Thank you, I've added a link to this comment to the follow-on Jira for updating the related docs content.